src/content/getusermedia/record/js/main.js

```diff
@@ -40,23 +45,11 @@ recordButton.addEventListener('click', () => {
 const playButton = document.querySelector('button#play');
 playButton.addEventListener('click', () => {
   const superBuffer = new Blob(recordedBlobs, {type: 'video/webm'});
+  recordedVideo.src = null;
+  recordedVideo.srcObject = null;
   recordedVideo.src = window.URL.createObjectURL(superBuffer);
```

Comment:
Does it work to set srcObject instead here, given orphis' quest to destroy createObjectUrl?
(I know createObjectURL(blob) is not being removed, but there's no reason to recommend createObjectURL in any way, shape or form.)